src/lycophron/tasks/tasks.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Lycophron is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""Lycophron tasks implementation."""
+
+from celery import group
+import os
+
+from . import app
+from ..db import db
+from ..client import create_session
+
+# TODO tasks are using 'db' directly. Breaks the basic flow Interface -> Business -> Data.
+# TODO tasks do not have any logging implemented yet (e.g. success, errors).
+# TODO tasks are not updating the record's local status (e.g. 'SUCCESS', 'FAILED', etc)
```

Comment:
This is important given that we do not have any means to assess whether a deposit was already created or not. We could be creating duplicates.